proposals/core.md

```diff
@@ -189,56 +282,82 @@ Below are some sample bidi commands that illustrate how message routing would wo
 ```json
 {
     "id": 0,
+    "to": "browsingContext/83c286bc-dc72-4678-a79f-c033b76498d7",
     "method": "navigateTo",
-    "params": { "browsingContextId": "<ID>", "url": "http://example.com" }
+    "params": { "url": "http://example.com" }
 }
 ```
 
-*Send command to script context*
+*Send command to JS realm*
 
 ```json
 {
     "id": 0,
+    "to": "realm/83c286bc-dc72-4678-a79f-c033b76498d7",
     "method": "executeSync",
     "params": {
-        "scriptContextId": "<ID>",
         "script": "return document.title;",
         "args": []
     }
 }
 ```
 
-#### Close a target
+#### Close a top-level window target
 
 ```json
 {
     "id": 0,
+    "to": "window/83c286bc-dc72-4678-a79f-c033b76498d7",
     "method": "closeTarget",

```

Comment:
This isn't new in the proposal, but I wonder if we really want closing to be generic? The only two things that can be closed AFAIK are windows and tabs. The whole browser itself would be closed by ending the session, I suppose.

So right now, window is the only type of target that can be closed at all.